db/migrate/20230526161454_create_actors_movies_join_table.rb

```diff
@@ -0,0 +1,8 @@
+class CreateActorsMoviesJoinTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :actor_movies, id: false do |t|
+      t.belongs_to :actor
```

Comment:
A key issue in your relationships could stem from this not being the right syntax for creating a joins table. Make sure you use the `rails g migration` to set up a migration. We should see something like this:
`t.references :movie, null: false, foreign_key: true`
`t.references :actor, null: false, foreign_key: true`